common/src/main/java/org/apache/rocketmq/common/attribute/AttributeParser.java

```diff
@@ -16,38 +16,44 @@
  */
 package org.apache.rocketmq.common.attribute;
 
-import com.google.common.base.Joiner;
 import com.google.common.base.Strings;
-
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class AttributeParser {
+
+    public static final String ATTR_ARRAY_SEPARATOR_COMMA = ",";
+
+    public static final String ATTR_ARRAY_SEPARATOR_EQUAL_SIGN = "=";
+
+    public static final String ATTR_ARRAY_SEPARATOR_PLUS_SIGN = "+";
+
+    private static final String ATTR_ARRAY_SEPARATOR_MINUS_SIGN = "-";
```

Comment:
In fact not all of these constants are separators, pls give them names showing the exact purpose of them, eg. : What's the meaning of 'plus' or 'minus' in the context of attribute parsing?
BTW, I'd prefer making these constant fields as `private`, for they are not referenced outside `AttributeParse`.

Reply:
> In fact not all of these constants are separators, pls give them names showing the exact purpose of them, eg. : What's the meaning of 'plus' or 'minus' in the context of attribute parsing? BTW, I'd prefer making these constant fields as `private`, for they are not referenced outside `AttributeParse`.

I see what you mean, thanks for you suggest, I will polish this code again.